hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -278,8 +282,8 @@ private boolean executeHttpOperation(final int retryCount,
       // dump the headers
       AbfsIoUtils.dumpHeadersToDebugLog("Request Headers",
           httpOperation.getConnection().getRequestProperties());
-      AbfsClientThrottlingIntercept.sendingRequest(operationType, abfsCounters);
-
+      intercept = AbfsClientThrottlingInterceptFactory.getInstance(accountName, isAutoThrottlingEnabled, isSingletonEnabled);
```

Comment:
this naming is not consistent with the one used below. In one place the variable is called `intercept` and in another `instance`.

Reply:
Fixed.